docs/rules/prefer-arrow-callback.md

```diff
@@ -37,6 +37,27 @@ foo(function() { return this.a; });
 foo(function bar(n) { return n && n + bar(n - 1); });
 ```
 
+## Options
+
+`prefer-arrow-callback` receives a single, optional argument which is an options object.
+
+### functionsWithThis
+
```

Comment:
I agree. I actually went with `dynamicThis` and then saw you suggested this name and reverted in my patch but obviously not in my commit message :)

How about `unboundThis`?